main.py

```diff
@@ -88,7 +88,28 @@ async def on_message(message: Message):
             await message.channel.send("You need to use !list first, sugar.")
             return
 
-        await command_play(message)
+        command_args = message.content.split()[1:]
+        skip_count = 0
+        if len(command_args) > 1:
+            bust_index = 0
```

Comment:
```suggestion
```

As `bust_index` is either set below or the function returns, this line isn't necessary.